utils/hct/hctdb.py

```diff
@@ -866,6 +866,11 @@ def populate_categories_and_models(self):
                 "closesthit",
                 "miss",
             )
+        for i in ("MaybeReorderThread").split(","):
+            self.name_idx[i].shader_stages = (
```

Comment:
Forgot category and shader model here:

```suggestion
        for i in ("MaybeReorderThread").split(","):
            self.name_idx[i].category = "Shader Execution Reordering"
            self.name_idx[i].shader_model = 6, 9
            self.name_idx[i].shader_stages = (
```

Note: generated files will need updating after this, so it should be applied offline.

Reply:
I just did this, since it was easy for me to do at this time.